cocos/renderer/gfx-metal/MTLDevice.mm

```diff
@@ -16,8 +16,10 @@
 #include "MTLTexture.h"
 #include "MTLTextureView.h"
 #include "MTLSampler.h"
-#import <MetalKit/MTKView.h>
 #include "MTLUtils.h"
+#include <platform/mac/CCView.h>
```

Comment:
Why need to include this header file?

Reply:
in `blitBuffer`, a mtlCommandQueue is required to create the blit command buffer